src/ansys/geometry/core/_grpc/_services/v0/repair_tools.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (C) 2023 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Module containing the repair tools service implementation (abstraction layer).
+
+This module defines an abstract base class for a gRPC-based repair tools service.
+The class provides a set of abstract methods for identifying and repairing various
+geometry issues, such as split edges, extra edges, duplicate faces, and more.
+"""
+
+from google.protobuf.wrappers_pb2 import BoolValue, DoubleValue
+import grpc
+
+from ansys.geometry.core.errors import protect_grpc
+
+from ..base.repair_tools import GRPCRepairToolsService
+
+
+class GRPCRepairToolsServiceV0(GRPCRepairToolsService):
+    @protect_grpc
+    def __init__(self, channel: grpc.Channel):
```

Comment:
You are missing the noqa statements, have a look at the other specific v0 implementations and docstrings. Otherwise pre-commit complains.

Reply:
after renewing my machine, I observed that my precommits are not updating much. Is there a specific setup that is needed to catch these small things locally before submit? I do precommit run before submitting, but it is not enforcing me to do so before commits as it was doing before.